Exercise_2.py

```diff
@@ -2,15 +2,25 @@
   
 # give you explanation for the approach
 def partition(arr,low,high):
-  
-  
     #write your code here
-  
+    l = low-1
+    pivot = arr[high]
+    for k in range(l, high):
+        if arr[k]<=pivot:
+            l +=1
+            arr[l],arr[k] = arr[k], arr[l]
+    arr[l],arr[high] = arr[high], arr[l]
+    return l
```

Comment:
In the end swap L+1 with high and return L+1, This will ensure everything to left is lesser than L+1 and everything to the right greater.